app/javascript/packs/searchParams.js

```diff
@@ -0,0 +1,220 @@
+/*global checkUserLoggedIn, showLoginModal, userData, buildArticleHTML, initializeReadingListIcons, initializeAllFollowButts, initializeUserFollowButts*/
+/*eslint no-undef: "error"*/
```

Comment:
Just a tiny cosmetic change:

```suggestion
/* global checkUserLoggedIn, showLoginModal, userData, buildArticleHTML, initializeReadingListIcons, initializeAllFollowButts, initializeUserFollowButts */
/* eslint no-undef: "error" */